src/util/RedirectUtil.js

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) 2015-2018, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(function () {
+  var fn = {};
+
+  /**
+   * Redirects the browser to a new URL by directly setting window.location.
+   */
+  fn.setWindowLocationTo = function (url) {
```

Comment:
- Prefer to keep naming convention `redirectTo` for window location change redirection because it's more common  perspective IMHO.
- And name the other more explicitly like `redirectToWithFormGet`

Reply:
Sure. I'll leave final naming up to you.